src/tools/rest-backend/rest-backend-config.json

```diff
@@ -0,0 +1,14 @@
+{
+	"service": {
+		"api": "http",
+		"port": 8080,
+		"ip": "0.0.0.0"
+	},
+	"http" : {
+		"script_names": ["/"]
+	},
+	"api_specification": {
```

Comment:
What do you mean? It is an option that allows to set the target link for the api blueprint (raw and pretty).